<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>17</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器ip</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器端口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonConnect">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditRead">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditWrite">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineIP">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePort">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSend">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonClose">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>disconnect</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
